terraform/modules/query-service-infra/bucket.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_s3_bucket" "lambda_deployments" {
+  bucket = "query-service-lambda-deployment-${var.deployment_stage}"
```

Comment:
Creating an s3 bucket just for a lambda artifact seems a little much. Can you use a single one and either namespace or hash the artifacts?

Reply:
👍